homeassistant/components/device_tracker/tado.py

```diff
@@ -26,15 +26,15 @@
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=30)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_USERNAME): cv.string,
     vol.Required(CONF_PASSWORD): cv.string,
-    vol.Required(CONF_USERNAME): cv.string
+    vol.Required(CONF_HOME_ID):  cv.string
```

Comment:
Would it be possible to make home id optional and default to the original `/me` url if no home id is given? Or even better, by default fetch all homes and allow an optional home id to filter results to a specific home.